### ISSUE TITLE
--tail option for drush watdchdog-show is missing in drush 9

### ISSUE THREAD

**Issue by zuernBernhard** [drush-ops/drush#3414]
drush ws --tail 

                                                          
  [Symfony\Component\Console\Exception\RuntimeException]  
  The "--tail" option does not exist.

**Comment by zuernBernhard** [comment]
Drush version : 9.1.0

**Comment by greg-1-anderson** [comment]
Some existing Drush commands were simplified in Drush 9. I think that `--tail` would still be a welcome addition, if someone wanted to submit a PR.

**Comment by thursdaybw** [comment]
I don't agree that removing valuable features counts as 'simplified'.

**Comment by thursdaybw** [comment]
To get around this, at least on linux, this command achieves a similar result
`watch -n 1 drush ws`

**Comment by olofjohansson** [comment]
Agreed. Tail is very valuable when developing since you can see hidden problems as they occur.

Tried https://stackoverflow.com/a/9574123 for macOS (similar to watch), but it doesn't cut it since it prints the 10 latest messages over and over again.

I'd really need tail in order to follow the log messages properly while developing.

**Comment by greg-1-anderson** [comment]
Valuable features require maintainers. The way to keep the functionality you need working in Open Source software is to contribute pull requests to improve the state of affairs.

For those who strongly value or need `ws --tail`, Drush 8 is still maintained, and the option is working there. Folks can stay on `8` until the features they need are available in master.

**Comment by serundeputy** [comment]
started on this @ DrupalConNA Nashville ... hope to finish this weekend.

**Comment by serundeputy** [comment]
filed a PR: https://github.com/drush-ops/drush/pull/3523

@greg-1-anderson I'd appreciate a review; the Circle CI builds are failing, but it does not seem related to this PR; if it is perhaps you can help me understand how it does.
